libs/Microsoft.MixedReality.WebRTC.Unity/Assets/Microsoft.MixedReality.WebRTC.Unity/Scripts/Media/RemoteAudioSource.cs

```diff
@@ -173,9 +332,73 @@ private void TrackRemoved(WebRTC.PeerConnection.TrackKind trackKind)
             }
         }
 
-        //private void RemoteAudioFrameReady(AudioFrame frame)
-        //{
-        //    FrameQueue.Enqueue(frame);
-        //}
+        private void EnqueueCreateAudioClipOnMainThread(int sampleRate, int channelCount)
+        {
+            Debug.Assert(_audioClipEnqueuedCreate == false);
+            _audioClipEnqueuedCreate = true;
+            _mainThreadWorkQueue.Enqueue(() =>
+            {
+                int lengthSamples = sampleRate / 10; // Buffer up to 100ms of data.
+                _audioClip = AudioClip.Create("RemoteClip", sampleRate, channelCount, sampleRate, true, _audioData.OnClipRead);
+                _audioClipEnqueuedCreate = false;
+                var aud = GetComponent<UnityEngine.AudioSource>();
+                if (aud)
+                {
+                    aud.clip = _audioClip;
+                    aud.volume = 1.0f;
+                    aud.loop = true;
+                    aud.bypassEffects = true;
```

Comment:
Arguable, no? What's the rationale for preventing the user from adding audio effects?